introduction/modules/modules.md

```diff
@@ -215,10 +218,10 @@ a resource in OpenZaak as an *eenvoudig informatie object* (simple information o
 SmartDocuments itself is an external service that deals with generating documents based on templates and placeholders.
 This module offers a plugin which can connect to this service to generate documents.
 
-### Wordpress mail
+### Wordpress email
 
-The wordpress mail plugin is a plugin for the Wordpress CMS that provides a REST API for sending mail. This module 
-includes an implementation of the mail module that uses a connector to send mail through this API.
+The Wordpress email plugin is a plugin for the Wordpress CMS that provides a REST API for sending emails. This module 
```

Comment:
I think I would rephrase this section to:

```### Wordpress mail

The Wordpress mail plugin is a plugin for the Wordpress CMS that provides a REST API for sending emails. This module
includes an implementation of the 'mail' module, which uses a connector to send emails through this API.```

Reply:
The module is called wordpress mail, so I'd like to suggest to stick to that name when talking about the module. Same goes for the mail module.